src/plugins/table/table.yml

```diff
@@ -0,0 +1,6 @@
+name:
+  en: Table
+  de: Tabelle
+renderMedia:
+  en: Render media
```

Comment:
Small thing: we have this now in 4 places, should we move it to `common.yml`?

Reply:
True, I've moved it there! Thanks!